policyengine_us/variables/gov/states/la/tax/income/exemptions/personal/la_dependents_exemption.py

```diff
@@ -0,0 +1,22 @@
+from policyengine_us.model_api import *
+
+
+class la_dependents_exemption(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Louisiana qualified dependents exemption"
+    reference = (
+        "https://www.revenue.louisiana.gov/taxforms/6935(11_02)F.pdf#page=1"
+    )
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.LA
+
+    def formula(tax_unit, period, parameters):
+        dependents = tax_unit("tax_unit_dependents", period)
+        return (
```

Comment:
similar for widow etc
```suggestion
        p = parameters(
            period
        ).gov.states.la.tax.income.exemptions
        return dependents * p.dependent
```